ui/apps/platform/src/Components/CompoundSearchFilter/attributes/policy.ts

```diff
@@ -16,4 +16,32 @@ export const Category: CompoundSearchFilterAttribute = {
     inputType: 'autocomplete',
 };
 
-export const policyAttributes = [Name, Category];
+export const EnforcementAction: CompoundSearchFilterAttribute = {
+    displayName: 'Enforcement action',
+    filterChipLabel: 'Enforcement action',
+    searchTerm: 'Enforcement',
+    inputType: 'select',
+    inputProps: {
+        options: [
+            { value: 'UNSET_ENFORCEMENT', label: 'Unset Enforcement' },
+            { value: 'SCALE_TO_ZERO_ENFORCEMENT', label: 'Scale to zero enforcement' },
+            {
+                value: 'UNSATISFIABLE_NODE_CONSTRAINT_ENFORCEMENT',
+                label: 'Unsatisfiable node constraint enforcement',
+            },
+            { value: 'KILL_POD_ENFORCEMENT', label: 'Kill pod enforcement' },
+            { value: 'FAIL_BUILD_ENFORCEMENT', label: 'Fail build enforcement' },
+            { value: 'FAIL_KUBE_REQUEST_ENFORCEMENT', label: 'Fail kube request enforcement' },
+            {
+                value: 'FAIL_DEPLOYMENT_CREATE_ENFORCEMENT',
+                label: 'Fail deployment create enforcement',
+            },
+            {
+                value: 'FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT',
+                label: 'Fail deployment update enforcement',
+            },
```

Comment:
Nits: 
- could we remove `"enforcement"` from the end of each label, it seems redundant with the other text?
- could "Unset" be the last option in the list?

Reply:
Or - could the options be alphabetical? In that case "unset" would naturally fall to the bottom.

Reply:
Sure, we can do that 👍🏼